repo/packages/C/cockroachdb/0/package.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
I think @ryadav88 may have been requesting that that info goes somewhere in this file, but that info is correct.